src/XmlConverter.php

```diff
@@ -220,7 +220,7 @@ private function domElementToArray(DOMElement $element): array
                 // For plain text, return string
                 case XML_CDATA_SECTION_NODE:
                 case XML_TEXT_NODE:
-                    if ('' !== trim($childElement->textContent)) {
+                    if (trim($childElement->textContent) !== '') {
```

Comment:
Here, can you please namespace the function `trim` to indicate that it's the root one as follows: `\trim()`. This helps when it comes to performance ;)